### PR TITLE
jsp23_fat trace update to help diagnose injection/interceptor issue

### LINKED ISSUE #12193: add trace to the servers under FAT com.ibm.ws.jsp.2.3_fat to help diagnose intermittant injection/interceptor issue

### PR DESCRIPTION
**Pull request by epj**
fixes #12193

### CODE REVIEW

**Comment by epj** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.jsp.2.3_fat

File: dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspServer/server.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <feature>jsp-2.3</feature>
     </featureManager>
     
-    <logging traceSpecification="*=info=enabled:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all" maxFileSize="32" maxFiles="24" traceFormat="BASIC"/> 
+    <logging traceSpecification="*=info=enabled:JCDI=all:com.ibm.ws.webbeans*=all:org.apache.webbeans*=all:org.jboss.weld*=all:com.ibm.ws.cdi*=all:com.ibm.ws.wswebcontainer*=all:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all" maxFileSize="32" maxFiles="24" traceFormat="BASIC"/>
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspServer/server.xml:19]: Why are we adding CDI traces to a server that does not enable CDI?

Comment (pmd1nh) [reply dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspServer/server.xml:19]: I don't think we use com.ibm.ws.wswebcontainer any more

Comment (epj) [reply dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspServer/server.xml]: wswebcontainer is removed

Comment (epj) [reply dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspServer/server.xml]: trace is now only added to jspCdiServer

**Comment by epj** [comment]
#libbybot

**Pull request merged**